src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSets.js

```diff
@@ -25,6 +26,8 @@ export const PermissionSets = (props) => {
   const intl = useIntl();
   const showCallout = useShowCallout();
 
+  const isPermissionsUpdated = location.state?.isUpdated;
```

Comment:
looks strange, I would suggest 2 ways
1. track path instead
2. or invalidate queries when mutation happened (IMO it's more preferable way)